minio/minio
# Include system errors in health report

## Description

Some examples of system errors:

- audit is enabled
- updatedb is installed

## Motivation and Context

Some packages/processes running on the minio node (like audit logging, updatedb, etc) can adversely impact the performance of the minio server. Capturing such information in the health report will help in automating detection of such issues through the Subnet Health dashboard.

## How to test this PR?

- Build `minio` with this PR and start the server(s) using the updated binary
- Build `mc` using the latest main branch of `madmin-go`
- Generate the health report for the cluster using `mc admin subnet health {alias}`
- Verify that
  - the report contains a new node `sys -> errors` which contains one sub-node each for every server in the cluster.
  - if any server of the cluster has audit logging enabled, the error is present under `sys -> errors`, mapped to the specific node's anonymized address.

## Types of changes

## Checklist:
